src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/cars/addDriverToCar")

```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/addDriverToCar")

```

checklist

Reply:
I do not understand. Should be "/drivers/addDriverToCar" ???

Reply:
<img width="571" alt="image" src="https://user-images.githubusercontent.com/68428976/161700451-6bff0f56-3eca-4e56-a94c-a9d8000048cb.png">
sorry, I should write `/cars/drivers/add`